src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -135,7 +135,7 @@ def get(self, _, project, event_id):
         # populate event data
         Event.objects.bind_nodes([event], 'data')
 
-        commits = self._get_commits(event.project, event.get_tag('sentry:release'))
+        commits = self._get_commits(event.project, event.group.first_release_id)
```

Comment:
This is going to incur a query to resolve. And it's awkward since it also bridges databases.

In this case, I'd suggest making it explicit rather than implicit and do:

```python
group = Group.objects.get(id=event.group_id)
```